toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
would it make sense to move `bootcustomizer.go`, `customizeboot.go`, and `defaultgrubutils.go` into their own packages so that the code is separated by the public interface?
I think it will be a useful package for tools other than the image customizer...

Reply:
Possibly at some point. But it might be worth doing that after ISO and verity is fixed, so we have a good idea of exactly what the public API should be.